housecleaning

mjamesruggiero opened the pull request:
* merging upstream changes from pivotal
* cleaning up our own stuff

HaydenSikh commented:
Are we still able to use KeyCastr?  Maybe it's just my workstation but even when in the started state it seems to be a NOP.  If others see the same, might be good to remove it from the install.

Other than that, looks good to me.  I like the new suite of default tools.

mjamesruggiero commented:
Removed keycastr. 

Noticed yesterday that it was broken on my test machine; Googled different solutions for about 30 minutes before I moved on. If we find a patch/workaround for post-Mavericks OSX we can easily add it back.
